pkg/config/config_template.yaml

```diff
@@ -1875,6 +1875,18 @@ api_key:
 #
 # jmx_reconnection_timeout: 60
 
+## @param jmx_statsd_client_queue_size - integer - optional - default: 4096
+## @env DD_JMX_STATSD_CLIENT_QUEUE_SIZE - integer - optional - default: 4096
+## Determines the size of the JMXFetch statds client queue (in number of elements).
+#
+# jmx_statsd_client_queue_size: 4096
```

Comment:
This is a purely internal setting, I don't think we should document it in the config_template. Could you add this documentation as a comment on `pkg/config/config.go` instead? Also, could you mention what `elements` mean in this context? (does each element represent a metric/service check?)

Reply:
Sounds good.